src/components/booking-details/PassengersListCard.test.tsx

```diff
@@ -0,0 +1,73 @@
+import { BookingDetailsData } from '@/types/booking';
+import { render, screen } from '@testing-library/react';
+import dayjs from 'dayjs';
+import PassengersListCard from './PassengersListCard';
+
+const mockBooking: BookingDetailsData = {
```

Comment:
Please move it to `src/msw/bookingDetailsMockData.ts` and reuse in all other tests